setup/flags.go

```diff
@@ -43,7 +43,8 @@ func ParseFlags(monolith bool) *config.Dendrite {
 	}
 
 	cfg, err := config.Load(*configPath, monolith)
-
+	// TODO: just for testing
```

Comment:
Let's nuke this if we can and just specify consent-based URL stuff in the consent section of the config.